app/src/main/res/values-ja/FirefoxReality_localpages.xml

```diff
@@ -12,15 +12,15 @@
     <!-- This string is used as the body of the Private Browsing page that is opened when the
     Private Browsing Mode button from the tray is clicked. -->
     <string name="private_browsing_body"><![CDATA[
-<p>プライベートブラウジング中、%1$s は以下を<strong>保存しません</strong>:</p>
+<p>%1$0.sプライベートウィンドウでは以下を<strong>保存しません</strong>:</p>
```

Comment:
Looks like `%1$s` got changed to `%1$0.s`? I thought we were only supporting `%1$s`?

Reply:
Yeah, it should be reverted, that's probably a typo